release_scripts/signassert.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+################################################################################
+#                                                                              #
+# Copyright (C) 2011-2015, Armory Technologies, Inc.                           #
+# Distributed under the GNU Affero General Public License (AGPL v3)            #
+# See LICENSE or http://www.gnu.org/licenses/agpl.html                         #
+#                                                                              #
+################################################################################
+
+import getpass
+import os
+import sys
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..'))
+
+from armoryengine.PyBtcWallet import *
+from armoryengine.ArmoryUtils import CLI_OPTIONS, CLI_ARGS
+from jasvet import ASv1CS, readSigBlock, verifySignature
+
+def signAssertFile(wltPath, assertFile):
+   wlt = PyBtcWallet().readWalletFile(wltPath)
+   if not wlt.hasAddr(signAddress):
+      print 'Supplied wallet does not have the correct signing key'
+      exit(1)
+
+   print 'Must unlock wallet to sign the assert file...'
+   while True:
+      passwd = SecureBinaryData(getpass.getpass('Wallet passphrase: '))
+      if not wlt.verifyPassphrase(passwd):
+         print 'Invalid passphrase!'
+         continue
```

Comment:
I don't see the advantage to limiting the number of attempts. The user can always Ctrl-C to quit the script, so it is not like they will get stuck running the script forever if they can't remember the password. And it is not like a website where the encrypted data is on the server, where rate limiting would prevent brute force attacks. But maybe there is another reason to limit the attempts that I don't see.